tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -251,6 +252,19 @@ async def set_permission(self, descriptor, state, origin):
         return await self.webdriver.bidi_session.permissions.set_permission(
             descriptor=descriptor, state=state, origin=origin)
 
+class WebDriverBidiWebExtensionsProtocolPart(WebExtensionsProtocolPart):
```

Comment:
I think in this file we should only implement BiDiWebExtensionProtocolPart using bidi_session